build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
   id 'java-gradle-plugin'
+  id 'java-library'
```

Comment:
Just curious: Why was this line added?

Reply:
Added it locally to test something but forgot to remove it. [Commit](https://github.com/sonatype-nexus-community/scan-gradle-plugin/pull/182/commits/4753ad0fc473a544d3765a435113d42fbc9ce72c)